jerry-core/parser/js/js-parser-util.c

```diff
@@ -923,7 +923,7 @@ parser_error_to_string (parser_error_t error) /**< error code */
     }
     case PARSER_ERR_NON_STRICT_ARG_DEFINITION:
     {
-      return "Non strict argument definition.";
+      return "Non-strict argument definition.";
```

Comment:
"use strict" is an ecmascript directive (which is grammatically correct in English). However, the "non-strict" expression is a different language construct and it's incorrect without the dash (btw it's used at 328 distinct places in the project and  in the test262 repo in this dashed form).